app/services/creators/capital_eligibility_creator.rb

```diff
@@ -27,7 +27,11 @@ def create_eligibility(ptc)
     end
 
     def lower_threshold
-      Threshold.value_for(:capital_lower, at: @assessment.submission_date)
+      if @assessment.level_of_representation == "controlled"
+        Threshold.value_for(:capital_lower_controlled, at: @assessment.submission_date)
+      else
+        Threshold.value_for(:capital_lower, at: @assessment.submission_date)
```

Comment:
not just specifically here, but should `capital_lower` be renamed to `capital_lower_cerificated` for clarity?

Reply:
I think you're right, it's will make things clearer.